ajax/libs/bricklayer/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "bricklayer",
+  "filename": "bricklayer.min.js",
+  "version": "0.2.22",
+  "description": "Lightweight cascading grid layout library",
+  "homepage": "http://ademilter.github.io/bricklayer/",
+  "keywords": [
+    "masonry",
+    "bricklayer",
+    "columns",
+    "grid",
+    "responsive"
+  ],
+  "demo": "http://ademilter.github.io/bricklayer/",
+  "npmName": "bricklayer",
+  "npmFileMap": [
+    {
+      "basePath": "dist/",
+      "files": [
+        "*"
```

Comment:
`bricklayer-node.js` is just for node.js. Others (`bricklayer.js` and `bricklayer.min.js`) are built for browsers. `package.json` shows the main file as `bricklayer-node.js`.